components/Chips/src/MDCChipView.m

```diff
@@ -744,11 +744,22 @@ - (CGSize)sizeForAccessoryViewWithMaxSize:(CGSize)maxSize {
 }
 
 - (CGRect)titleLabelFrame {
-  CGRect imageFrame = CGRectUnion(_imageView.frame, _selectedImageView.frame);
+  // Default to the unselected image, but account for the selected image if it's shown.
+  CGRect imageFrame = _imageView.frame;
+  if (self.showSelectedImageView) {
```

Comment:
Do we need to do anything if `showSelectedImageView` is false?

Reply:
In that case, the default behavior would be to use the frame of `_imageView` (above).